tonami/Utterance.py

```diff
@@ -14,6 +14,7 @@
 # the pre-processing and can be passed to our classifier or visualization model
 # possibly stores its own classification/probability as well?
 
+DB_THRESHOLD = 10
```

Comment:
think you should add comment abt how this is arbitrary & can be tuned depending on environment